src/pages/TradeV3/perps/components/CollateralPanel.tsx

```diff
@@ -206,177 +101,59 @@ const Accounts: FC<{ isSolAccount: boolean }> = ({ isSolAccount }) => {
-            positions or increase your position size. This is calculated based on your Balance and the margin
-            requirements for the specific assets you are trading{' '}
+      </div>
+    )
+  }
+  return (
+    <div tw="flex items-center justify-between mt-2">
+      <div>
+        <div tw="flex items-center">
+          <img src={`/img/assets/healthIcon${mode}.svg`} alt="heart-icon" tw="h-5 w-5 mr-1.5" />
+          <InfoLabel>Health</InfoLabel>
+          <Tooltip>
+            <TooltipTrigger>
+              <div className={cn('ml-1.5')}>
+                <InfoImage mode={mode} />
+              </div>
+            </TooltipTrigger>
+            <TooltipContent>The health bar shows how close you are to being liquidated. </TooltipContent>
           </Tooltip>
         </div>
-        <span className="value">{formatNumberInThousands(Number(traderInfo.marginAvailable))} $</span>
-      </ACCOUNT_ROW>
-      {isSolAccount && (
-        <ACCOUNT_ROW $height={height}>
-          <div className="tooltip-row">
-            <span className="key">{ask + ' Liquidation Price'}</span>
-            <Tooltip color={mode === 'dark' ? '#F7F0FD' : '#1C1C1C'}>
-              The Liquidation Price is the price at which your position will be automatically closed out by the
-              trading platform if your margin falls below a certain threshold. The Liquidation Price is calculated
-              based on the current market price, your position size, and the margin requirements for the specific
-              assets you are trading.{' '}
-            </Tooltip>
-          </div>
-          <span className="value">{formatNumberInThousands(Number(traderInfo.liquidationPrice))} $</span>
-        </ACCOUNT_ROW>
-      )}
-    </WRAPPER>
-  )
-}
-
-const Fees = () => {
-  const feesInfo = PERPS_FEES
-  const currentTier = feesInfo[1]
-  const { height } = useWindowSize()
-  return (
-    <FEES $height={height}>
-      <div className="tier">
-        My current tier:{' '}
-        <span>
-          {currentTier.taker} Taker / {currentTier.maker} Maker
-        </span>
       </div>
-      <div className="disclaimer">*Disclaimer: Perps fees for GooseFx are subject to change.</div>
-      {feesInfo &&
-        feesInfo.map((item, index) => (
-          <Row key={index}>
-            <Col className={index === 0 ? 'spacing value' : 'value'} span={4}>
-              {item.tier}
-            </Col>
-            <Col className={index === 0 ? 'spacing tier-info value' : 'tier-info value'} span={12}>
-              {item.stake}
-            </Col>
-            <Col className={index === 0 ? 'spacing tier-info value' : 'tier-info value'} span={4}>
-              {item.taker}
-            </Col>
-            <Col className={index === 0 ? 'spacing tier-info value' : 'tier-info value'} span={4}>
-              {item.maker}
-            </Col>
-          </Row>
-        ))}
-    </FEES>
+      <div className="flex">
+        <div>{getBarsData()}</div>
```

Comment:
these functions would get called every render, ideally instead of making them a callback make them a memo

Reply:
useMemo is better option, done